README.md

```diff
@@ -34,23 +53,122 @@ They are said to be "bound" to the unit they are currently operating on.
+| [lpu](#lpu)  | Search for the nearest purine to the left   | |
+
+#### pun
+
+Not an Amino acid/instruction, but deserves special clarification. 
+
+Punctuates the strand, allowing one strand to encode for two or more separate enzymes. 
+When determining the enzymes encoded for by a strand, the prescence of the pun instruction 
+ends the enzyme early, and an additional enzyme is determined from the remaining strand, decoding 
+until the end of the strand or another pun is reached. These enzymes will act sequentially on
+the strand, one at a time. The portion of a strand that encodes for a specific enzyme is known
+as a 'gene', so one strand may contain one or more genes that encode for the same number of enzymes. 
+Pun has no folding direction, and therefore does not contribute to the folding pattern of the enzyme, 
+as well as no additional effects on the enzyme past encoding. 
+ 
+#### cut
+
+Cuts the strand at the unit the enzyme is currently bound to.
+The cut is placed to the right of the bound unit.
+Cut applies to both the "active" strand and the "complementary" strand that
+is being generated from the copy mode.
+
+#### del 
+
+Deletes the base that the enzyme is attached to, and then the enzyme moves to the right. 
+Only pertains to the active strand, *not* the complementary strand. 
+
+#### swi
+
+Switches the enzyme from the "active" strand to the "complementary" strand. Since the complementary strand
+is represented upside-down above the active strand (see [cop](#cop)), switching strands seems to reverse the order 
+of the bases in the now active strand. If a switch occurs and there are no bases in the complementary strand above 
+where the enzyme is acting, the enzyme falls off and ends acting.  
+
+#### mvr
+
+Moves the enzyme one base to the right. If the enzyme moves off the end of the strand or into a gap with no bases, then it finishes executing
+even if it had more instructions left to execute. 
+
+#### mvl
+
+Moves the enzyme one base to the left. If the enzyme moves off the front of the strand or into a gap with no bases, then it finishes executing
+even if it had more instructions left to execute, same as in mvr. 
+
+#### cop
+
+Places the complement of the current base into the complementary strand, and enters "copy mode." Each base has its corresponding
+complement, known as the complementary base pairin (see [Definitions](#Definitions)). Copy mode remains in effect until an off 
+instruction is reached (or the end of the strand/enzyme). While in copy mode, any base that is "touched" by the enzyme will 
+have its complementary base placed above it in the complementary strand. When bases are copied into the complementary 
+strand, they are placed *upside-down* above the element copied, so that the strands are mirrored to each other. Copy mode applies 
+after a move, so that the movement off of a strand or into a gap between bases still preserves the behavior of the strand ending normally.  
```

Comment:
```suggestion
*after* a move, so that the movement off of a strand or into a gap between bases still preserves the behavior of the strand ending normally.  
```
I think extra emphasis on the fact copy mode is applied after is important.